CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.15 [2023-04-21]
+
+_Bug fixes_
+
+- Fix queries that query gcp_project table to use project_id instead of name column as common dimensions. ([#96](https://github.com/turbot/steampipe-mod-gcp-compliance/pull/96))
```

Comment:
```suggestion
- Fixed the following queries to use the `project_id` column instead of the `name` column for `project` common dimension: ([#96](https://github.com/turbot/steampipe-mod-gcp-compliance/pull/96))
  - `logging_metric_alert_audit_configuration_changes`
  - `logging_metric_alert_custom_role_changes `
  - ``

```